src/main/java/io/specto/hoverfly/junit/dsl/SingleQuoteHttpBodyConverter.java

```diff
@@ -0,0 +1,56 @@
+package io.specto.hoverfly.junit.dsl;
+
+public class SingleQuoteHttpBodyConverter implements HttpBodyConverter {
+
+    private String convertedBody;
+
+    public SingleQuoteHttpBodyConverter(String json) {
```

Comment:
I thought the same but maybe some user want to use it directly? This was basically my concern.